### PR TITLE
SOV-3504 ms address for token extraction

### PR DESCRIPTION
**Pull request by dharkmattr**

### CODE REVIEW

File: contracts/masset/Masset.sol [hunk 1]
```diff
@@ -549,20 +555,42 @@ contract Masset is IMasset, IERC777Recipient, InitializableOwnable, Initializabl
...
+
+    /**
+     * @dev This can be called by a pre-defined mutisig address in order to 
+     *      convert some funds in one tx
+     * @param _tokenA     Address of the first bAsset
+     * @param _tokenB     Address of the second bAsset
+     * @param _amount     Amount to extract
+     */
+    function convertTokens(address _tokenA, address _tokenB, uint256 _amount) public {
+        require(msg.sender == adminMultisig, "not allowed");
```
Comment (tjcloa) [inline contracts/masset/Masset.sol:574]: i recommend to add requirement tokenA != tokenB

File: test/TestMasset.spec.ts [hunk 2]
```diff
@@ -55,6 +56,61 @@ contract("Masset", async (accounts) => {
         });
     });
```
Comment (tjcloa) [inline test/TestMasset.spec.ts:58]: missing tests for `Masset.convertTokens(...)`

**Review by tjcloa — CHANGES_REQUESTED** [review]
1) fix a failing CI check
2) pls restructure the repo to comply with Sovryn structure if no objections: rename `main` to `development` branch and create `main` branch used as target for versioning and immunify bug bounty programs
3) address inline comments

**Review by tjcloa — APPROVED** [review]

**Review by tjcloa — APPROVED** [review]

File: contracts/masset/Masset.sol [hunk 3]
```diff
@@ -549,20 +555,45 @@ contract Masset is IMasset, IERC777Recipient, InitializableOwnable, Initializabl
...
+    // admin function to extract tokens to pre-defined multisig address
+
+    /**
+     * @dev This can be called by a pre-defined mutisig address in order to 
+     *      convert some funds in one tx
+     * @param _tokenA     Address of the first bAsset
+     * @param _tokenB     Address of the second bAsset
+     * @param _amount     Amount to extract
+     */
+    function convertTokens(address _tokenA, address _tokenB, uint256 _amount) public {
```
Comment (koirikivi) [inline contracts/masset/Masset.sol]: - This is not non-reentrant. Whether it matters or not, I don't know.
- Why public and not external?

**Review by koirikivi — APPROVED** [review]
LGTM, but added one review comment nevertheless.

File: contracts/masset/Masset.sol (hunk 3 above)
Comment (dharkmattr) [reply contracts/masset/Masset.sol]: will change to external

Comment (dharkmattr) [reply contracts/masset/Masset.sol]: what do you mean by non-reentrant? It can only be called by the MS

**Review by koirikivi — APPROVED** [review]

**Pull request merged**